packages/trie/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 5.0.3 - 2023-02-16
+
```

Comment:
Do we need to mention the readme updates from #2525.  I can't remember if you normally include doc changes within the Changelog or not (since it's not a functional change to the code).

Reply:
I guess that's not so important, will skip, these small changes will just make its way into the code docs organically and hopefully help users a bit over time. There are no absolute "rules" what to include or not to include in the CHANGELOGs, these should just provide a helpful overview on what's new and relevant and might need some attention.